packages/chaire-lib-backend/src/utils/processManagers/TrRoutingProcessManager.ts

```diff
@@ -81,10 +80,28 @@ const startTrRoutingProcess = async (
     return processStatus;
 };
 
+const getCacheAllScenarios = () => {
+    // FIXME Should be simply `return
+    // config.trRouting?.single?.cacheAllScenarios === undefined ? false :
+    // config.trRouting.single.cacheAllScenarios;`, but we need to handle
+    // deprecation of `trRoutingCacheAllScenarios` value. Deprecated config
+    // option has precedence as it means it was already set by the user. The
```

Comment:
It's an edge case,but if the user have set both the old config setting and the new one I think that the new one should have precedence.